localstack/ext/java/pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>cloud.localstack</groupId>
     <artifactId>localstack-utils</artifactId>
     <packaging>jar</packaging>
-    <version>0.1.13</version>
+    <version>0.1.14-SNAPSHOT</version>
```

Comment:
We could change this to `0.1.14` - we're not typically creating snapshot releases.